ui/src/app/pages/analysis/leonardo-gce-launcher.tsx

```diff
@@ -380,9 +380,7 @@ export const LeonardoAppLauncher = fp.flow(
     }
 
     private isCreatingNewNotebook() {
-      const creating = parseQueryParams(this.props.location.search).get(
-        'creating'
-      );
+      const creating = parseQueryParams().get('creating');
```

Comment:
confirmed that notebook creation (vs. not) and playground mode (vs. not) continue to work, and Python and R kernel creation works